app/services/publisher_payout_feedback_status_updater.rb

```diff
@@ -0,0 +1,29 @@
+# typed: ignore
+
+class PublisherPayoutFeedbackStatusUpdater < BuilderBaseService
+  def self.build
+    new
+  end
+
+  def call(to_update_list:)
+    by_publisher = to_update_list.group_by { |i| i["owner"] }
+    by_publisher.each do |publisher_str, channels|
+      Rails.logger.info("Processing #{publisher_str} with channels: #{channels.map { |c| c["publisher"] }.join(";")}")
+      publisher = Publisher.where(id: publisher_str.sub("publishers#uuid:", "")).first
+      if publisher.blank?
+        Rails.logger.info("Couldn't find publisher #{publisher_str}!")
+        next
+      end
+      publisher_wp_id = publisher.selected_wallet_provider&.wallet_provider_id
+      provided_wp_id = channels[0]["walletProviderId"]
```

Comment:
Is this the channel's address?  I think that _should_ be different from the publisher's address, right? Just want to make sure I understand what we're checking.